### DISCUSSION TITLE
formState errors

### DISCUSSION THREAD

**Discussion by ghost** [react-hook-form/react-hook-form#10968] [Q&A] [answered]
I have a situation where I want to show that the field is faulty by default without even the user interacting with it or submitting the form, so that when he enters the page it would be marked as red(required) until the user chooses an option. Now I have a lot of fields that I want to behave this way, and I don't wanna use the !formState.isValid since it would mark all fields as faulty even if one of them isn't anymore. Also, using formState.errors.fieldName won't fix my problem since it only gets faulty once the user submit the form or starts interacting with the field.

**Comment by Moshyfawn** [comment, marked as answer]
```js
const { errors } = formState

useEffect(() => {
  trigger(["field1", "field2"])
}, [trigger])

errors.field1
```

**Reply by ghost** [reply]
Thanks, this worked!
